src/tools/pub/cmd/pub-cache.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: pub cache
-description: Use pub cache to manage your system cache.
+title: dart pub cache
+description: Use dart pub cache to manage your system cache.
 toc: false
 ---
 
 _Cache_ is one of the commands of the [pub tool](/tools/pub/cmd).
 
```

Comment:
Same comment as for pub-upgrade: we should explicitly talk about `pub` vs. `dart pub`.